samples/cocoapods/build.sh

```diff
@@ -0,0 +1,16 @@
+#!/usr/bin/env bash
+
+DIR=$(cd "$(dirname "${BASH_SOURCE[0]}" )" && pwd )
```

Comment:
For me it fails with 
``` Undefined symbols for architecture i386:
  "_OBJC_CLASS_$_Kotlin_libraryKotlinLibKt", referenced from:
      objc-class-ref in ViewController.o
ld: symbol(s) not found for architecture i386
```

Reply:
Specified an architecture explicitly in the build.sh.